libraries/webFrontend.js

```diff
@@ -143,8 +151,8 @@ exports.printFolder = function (objects, objectsPath, debug, objectInterfaceName
         var datExists = fs.existsSync(path.join(objectsPath, objectKey, identityFolderName, '/target/target.dat'));
         var xmlExists = fs.existsSync(path.join(objectsPath, objectKey, identityFolderName, '/target/target.xml'));
         var jpgExists = fs.existsSync(path.join(objectsPath, objectKey, identityFolderName, '/target/target.jpg'));
-
-        if (xmlExists && (datExists || jpgExists)) {
+        
+        if ((xmlExists && datExists && jpgExists) || (xmlExists && jpgExists)) {
```

Comment:
This is equivalent to xmlExists && jpgExists since if the first is true the second will also be true

Reply:
Any of these files can exist or not exist. Two states are allowed:
All files exist or xml and jpg exist.
There is no state where xml and dat is allowed.
The previous expression produced fault results.

Reply:
There's just no reason to have the first half of this expression since if (xmlExists && datExists && jpgExists) is true then (xmlExists && jpgExists) is always also true

Reply:
Sure so we could write it like: 
if (xmlExists && ((datExists && jpgExists)  || jpgExists))

Reply:
No it just simplifies to `xmlExists && jpgExists`

When all files exist the xml and jpg files exist